acceptance/openstack/blockstorage/v3/volumetypes_test.go

```diff
@@ -55,3 +55,57 @@ func TestVolumeTypes(t *testing.T) {
 	th.AssertEquals(t, description, newVT.Description)
 	th.AssertEquals(t, isPublic, newVT.IsPublic)
 }
+
+func TestvolumeTypesExtraSpecs(t *testing.T) {
```

Comment:
It doesn't look like this test is being run. I think it might have to do with the lower-case "v".

Reply:
done